src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$color-text-primary: #060b35;
+$color-text-secondary: #616070;
+
+$color-accent-blue: #00acdc;
+$color-white-smoke: #f3f3f3;
+$color-white: #fff;
```

Comment:
You should not name variables by the color that is stored there, because the color can change during the development process and you will have to update this not only in the variable definition, but also throughout the code.
Therefore, it is worth naming variables based only on their purpose.

Reply:
Thanks, that makes sense, now I understand.